src/python/pants/engine/internals/graph_test.py

```diff
@@ -1586,8 +1586,12 @@ class ChildTarget(Target):
 @pytest.mark.parametrize(
     "field_content",
     [
-        "tagz=['tag']",
-        "tagz=parametrize(['tag1'], ['tag2'])",
+        "tagz=('tag',)",
```

Comment:
Note that we can still pass a list here, and the right thing will happen, but the test will then fail on a mismatching expected error message, as the kwarg will have been tuplelized.